cwms-data-api/src/main/java/cwms/cda/api/LocationGroupController.java

```diff
@@ -235,31 +235,32 @@ public void create(@NotNull Context ctx) {
                 + "unassign all existing locations before assigning new locations specified in the content body "
                 + "Default: false"),
             @OpenApiParam(name = OFFICE, required = true, description = "Specifies the "
-                + "owning office of the location group to be updated"),
+                + "user office making the request"),
```

Comment:
We probably want to make this as clear as we can. It is the office used for the owning category, group and location with the caveat that for CWMS groups it is for the location only.

Reply:
The description field should also get updated to describe all the operations supported by this endpoint.

Reply:
It also seems to break the several concept we have used through CDA, though so does having an object like this owned by "CWMS" so I'm not sure there's a better way that reusing the office field in this way. 

Alternatively instead of the user needs to specifiy this here, for at least the case of assigning location, or time series in the other, it may make more sense to process them one at a time as one could provide a list of locations/timeseries each with a different office they have permissions to. Perhaps not likely now, but I could see it becoming a thing.

Reply:
The loc_alias_t is missing the office id component that would make this more explicit, the thing we thought of was that CDA could start that move by updating the location-id to a CwmsId and then iterate one by one. That would be a backwards incompatible change though given the JSON format restructure.

Reply:
I added more detail to both the endpoint description and the parameter description.

Reply:
Can you add a ticket to address the above? Needing the location office to get included that's separate from the category/group office.

Reply:
I've created a DB ticket here: [https://jira.hecdev.net/browse/CWDB-308](https://jira.hecdev.net/browse/CWDB-308)